semeio/workflows/localisation/localisation_config.py

```diff
@@ -149,6 +233,74 @@ class ExponentialConfig(GaussianConfig):
     method: Literal["exponential_decay"]
 
 
+class ScalingFromFileConfig(BaseModel):
+    method: Literal["from_file"]
+    filename: str
+    param_name: str
+
+
+class ScalingForSegmentsConfig(BaseModel):
+    method: Literal["segment"]
+    segment_filename: str
+    param_name: str
+    active_segments: conlist(item_type=conint(ge=0))
+    scalingfactors: conlist(item_type=confloat(ge=0.0))
+    smooth_ranges: Optional[conlist(item_type=conint(ge=0), min_items=2, max_items=2)]
+
+    @validator("active_segments", pre=True)
+    def validate_active_segments(cls, values):
+
+        if isinstance(values, int):
+            active_segments = [values]
+        elif isinstance(values, list):
+            active_segments = values
+            for v in active_segments:
+                if v < 0:
+                    raise ValueError(
+                        "Active segments must be integers >=0"
+                        " in keyword 'active_segments.'"
+                    )
+        return active_segments
+
+    @validator("scalingfactors", pre=True)
+    def validate_scalingfactors(cls, values):
```

Comment:
Not needed, use:
```
    scalingfactors: conlist(item_type=confloat(ge=0.0, le=1.0))
```

Reply:
Are there any other way to specify better error messages than what is autogenerated by pydantic?